infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -19,6 +19,27 @@ LLVM_DEP_PACKAGES="build-essential make cmake ninja-build git subversion python2
 apt-get install -y $LLVM_DEP_PACKAGES
 
 # Checkout
+CHECKOUT_RETRIES=10
```

Comment:
That seems too much, how about 3 ?

Reply:
Actually, I'd like to put even bigger value. If we fail here, we're gonna stuck with old base images for one more day (unless someone manually runs the build). IMO we should try as hard as we can, on a reasonable scale of course.